> exec `make generate-openapi` and commit

... again

also web.lint:
```
/woodpecker/src/github.com/woodpecker-ci/woodpecker/web/src/views/repo/pipeline/PipelineWrapper.vue
  90:32  warning  Expected no line breaks before closing bracket, but 1 line break found  vue/html-closing-bracket-newline
```

also web.typecheck:
```

src/views/repo/pipeline/PipelineWrapper.vue(81,61): error TS2339: Property 'cancel_info' does not exist on type 'Pipeline'.
src/views/repo/pipeline/PipelineWrapper.vue(84,38): error TS2339: Property 'cancel_info' does not exist on type 'Pipeline'.
src/views/repo/pipeline/PipelineWrapper.vue(86,78): error TS2339: Property 'cancel_info' does not exist on type 'Pipeline'.
src/views/repo/pipeline/PipelineWrapper.vue(89,88): error TS2339: Property 'cancel_info' does not exist on type 'Pipeline'.
src/views/repo/pipeline/PipelineWrapper.vue(93,43): error TS2339: Property 'cancel_info' does not exist on type 'Pipeline'.
src/views/repo/pipeline/PipelineWrapper.vue(94,84): error TS2339: Property 'cancel_info' does not exist on type 'Pipeline'.
 ELIFECYCLE  Command failed with exit code 2.
```

--- a/server/model/pipeline.go
+++ b/server/model/pipeline.go
@@ -89,5 +89,5 @@ type PipelineOptions struct {
 
 type CancelInfo struct {
 	CanceledByUser string `json:"canceled_by_user,omitempty"`
-	SupersededBy int64  `json:"superseded_by,omitempty"`
+	SupersededBy   int64  `json:"superseded_by,omitempty"`
 } //	@name	CancelInfo

--- a/web/src/lib/api/types/pipeline.ts
+++ b/web/src/lib/api/types/pipeline.ts
@@ -8,8 +8,8 @@ export interface PipelineError<D = unknown> {
 }
 
 export interface CancelInfo {
-  reason: string;
-  data?: Record<string, string>;
+  canceled_by_user: string;
+  superseded_by: number
 }
 
 // A pipeline for a repository.
@@ -86,8 +86,6 @@ export interface Pipeline {
   reviewed_by: string;
 
   reviewed: number;
-
-  cancel_info?: CancelInfo;
 
   // The steps associated with this pipeline.
   // A pipeline will have multiple steps if a matrix pipeline was used or if a rebuild was requested.

--- a/web/src/lib/api/types/pipeline.ts
+++ b/web/src/lib/api/types/pipeline.ts
@@ -92,6 +92,8 @@ export interface Pipeline {
   workflows?: PipelineWorkflow[];
 
   changed_files?: string[];
+
+  cancel_info: CancelInfo;
 }
 
 export type PipelineStatus =

--- a/web/src/lib/api/types/pipeline.ts
+++ b/web/src/lib/api/types/pipeline.ts
@@ -9,7 +9,7 @@ export interface PipelineError<D = unknown> {
 
 export interface CancelInfo {
   canceled_by_user: string;
-  superseded_by: number
+  superseded_by: number;
 }
 
 // A pipeline for a repository.

--- a/web/src/views/repo/pipeline/PipelineWrapper.vue
+++ b/web/src/views/repo/pipeline/PipelineWrapper.vue
@@ -87,8 +87,7 @@
                 class="hover:underline"
                 >{{
                   $t('repo.pipeline.cancel_info.superseded_by', { pipelineId: pipeline.cancel_info.superseded_by })
-                }}</router-link
-              >
+                }}</router-link>
             </template>
             <template v-else-if="pipeline.cancel_info.canceled_by_user">
               {{ $t('repo.pipeline.cancel_info.canceled_by_user', { user: pipeline.cancel_info.canceled_by_user }) }}

